<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FenetreProgrammerTache</class>
 <widget class="QDialog" name="FenetreProgrammerTache">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="lProjet">
     <property name="text">
      <string>Projet :</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lDateHeure">
     <property name="text">
      <string>Date &amp; heure :</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QDialogButtonBox" name="bCancel">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="inProjet"/>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="inTache"/>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="inTache2"/>
   </item>
   <item row="4" column="1">
    <widget class="QDateTimeEdit" name="inDateHeure"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lTache">
     <property name="text">
      <string>Tache :</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QDialogButtonBox" name="bOK">
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>bCancel</sender>
   <signal>accepted()</signal>
   <receiver>FenetreProgrammerTache</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bCancel</sender>
   <signal>rejected()</signal>
   <receiver>FenetreProgrammerTache</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
